Core/Core/Api/GraphQL/Client.Subscriptions.Commit.cs

```diff
@@ -42,8 +42,7 @@ public void SubscribeCommitCreated(string streamId)
       }
       catch (Exception e)
       {
-        Log.CaptureException(e);
-        throw e;
+        Log.CaptureAndThrow(e);
```

Comment:
Was this one missed? There's a few more below...

Reply:
in what way? i'm using `CaptureAndThrow` here on purpose if that's what you mean